src/util.ts

```diff
@@ -100,9 +100,9 @@ export function getManifests(application: string): Promise<string[]> {
     : getManifestsFromSideloadingDirectory(application);
 }
 
-export function addManifest(application: string, manifestPath: string): Promise<any> {
+export function addManifest(application: string, parsedGuid: string, manifestPath: string): Promise<any> {
```

Comment:
`manifestId` would be a better name instead of `parsedGuid`. Please update in all of the relevant places.

Reply:
Fixed